layouts/partials/footer.html

```diff
@@ -13,7 +13,9 @@
 <p id="back-to-top">
   <a class="visible-xs" href="#top">{{ i18n "navigation-back-to-top" }}</a>
 </p>
-<footer id="solstice-footer">
```

Comment:
Did you make sure that we don't have any styles attached to this id?

Reply:
That's a good point, thx for pointing out.
I add the id back when no footer params.

```
{{- with .Params.footer_class | default .Site.Params.footer_class | default "solstice-footer"}} 
<footer class="{{ . }}" id="{{ . }}">
{{ end }}
```

Reply:
we should probably not tie the class to the ID. We can have multiple words for a class, like where we would want to use mostly standard solstice-footer styling, but add a class with a single override (as an example). We should make this 2 separate params

Reply:
We need to manage the id and class separately. 

You can only have 1 ID but you might want to add multiple classes.

How about this:

`<footer class="{{- .Params.footer_id | default .Site.Params.footer_id | default "solstice-footer"}}" id="{{- .Params.footer_class | default .Site.Params.footer_class | default "solstice-footer"}}">`